api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderConstants.java

```diff
@@ -134,6 +134,7 @@
         public static final String VITALS_CAPTURE_VISIT_ACTION = "pih.form.vitals";
         public static final String CONSULT_NOTE_VISIT_ACTION = "pih.form.consult";
         public static final String CONSULT_NOTE_INITIAL_VISIT_ACTION = "pih.form.consultInitial";
+		public static final String NURSE_CONSULT_NOTE_VISIT_ACTION = "pih.form.nurseConsult";
```

Comment:
static constant used to name the new "consult initial" button we are adding under visit actions